eventsourcing/src/test/java/org/axonframework/eventsourcing/eventstore/SimpleEventStoreTest.java

```diff
@@ -319,4 +320,9 @@ private void doConsumeAll(MessageStream<? extends EventMessage<?>>... sources) {
             throw assertionFailedError;
         }
     }
+
+    @Nonnull
+    private static UnitOfWork aUnitOfWork() {
```

Comment:
Nit: Perhaps we can make a test utils from this? That would allow us a spot to clearly document that `UnitOfWork` instances constructed in this flow **DO NOT** have a workable `ApplicationContext` in them.

Added, it would allow us a single `INSTANCE` constant of the `SimpleUnitOfWorkFactory` instead of creating one every time.